py/modbuiltins.c

```diff
@@ -52,6 +52,11 @@ STATIC mp_obj_t mp_builtin___build_class__(size_t n_args, const mp_obj_t *args)
     // set the new classes __locals__ object
     mp_obj_dict_t *old_locals = mp_locals_get();
     mp_obj_t class_locals = mp_obj_new_dict(0);
+    #if MICROPY_PY_CLASS_ORDERED_LOCALS
+    // keep the locals ordered
```

Comment:
Nitpick, but this looks superfluous: it's in a block 'class ordered locals' and sets the `is_ordered` flag